src/operations.ts

```diff
@@ -1,3 +1,14 @@
-export const DONE = "done";
-export const NAME = "name";
-export const RESPONSE = "response";
+const DONE = "done";
+const NAME = "name";
+const RESPONSE = "response";
+import { SignifyClient } from "signify-ts";
+
+export const waitKeriaOp = async (client: SignifyClient, op: any, log: boolean = false) => {
```

Comment:
I'd like to have consistent naming for exported values/functions

Reply:
should we call thisc`\getKeriaOperationResult`or?